script/test

```diff
@@ -55,19 +55,26 @@ readonly TOP_BUILDDIR="${BUILD_DIR}/otbr"
 #######################################
 print_result() {
     local exit_code=0
+    local QUIET=${QUIET:1}
```

Comment:
```suggestion
    local quiet=${QUIET:1}
```

Reply:
How about `readonly QUIET=${QUEIT:1}`?

Reply:
If I am not wrong QUIET is an environment var passed by used. Is it a good a idea to override a name here?

it seems you are using lowercase for local variables and uppercase for global variables
```bash
local exit_code=0
```

I would prefer `local quiet=${quiet:1}` if it does not need to be global.

Reply:
QUIET is a constant, so uppercase is proper. And it should be accessible throughout the script in case other functions will need this too. Thoughts?

Reply:
then move `readonly QUIET=${QUIET:1}` to the beginning of this script.